generator/utils.hpp

```diff
@@ -1,8 +1,27 @@
 #pragma once
 
 #include "indexer/index.hpp"
+#include "indexer/mwm_set.hpp"
+
+#include "platform/local_country_file.hpp"
+
+#include <string>
 
 namespace generator
 {
+class SingleMwmIndex
+{
+public:
+  explicit SingleMwmIndex(std::string const & mwmPath);
+  Index & GetIndex() { return m_index; }
```

Comment:
Please, add a blank line between ctor and methods.

Reply:
Done